File: CSharp/AverageOfAllElementsAsWholeNumber.cs
using System.Linq;

namespace CSharp
{
    // Create a function that takes an array as an argument and returns true or false 
    // depending on whether the average of all elements in the array is a whole number or not.
    // https://edabit.com/challenge/mDNt9RaNdcZwj6xNX
    public static class AverageOfAllElementsAsWholeNumber
    {
        public static bool IsAvgWhole(int[] arr) => arr.Sum() % arr.Length == 0;
    }
}

File: js/averageOfAllElementsAsWholeNumber.js
// Create a function that takes an array as an argument and returns true or false
// depending on whether the average of all elements in the array is a whole number or not.
// https://edabit.com/challenge/arzsQcGsstJxHAnGi

const isAvgWhole = (arr) =>
  arr.reduce((acc, curr) => acc + curr, 0) % arr.length === 0;

module.exports = isAvgWhole;
